docs/source/scripts_and_their_functions.rst

```diff
@@ -31,4 +31,12 @@ EBU-TT-D Encoder
 This script is an extension of simple consumer and is responsible for resegmenting and converting the incoming
 EBU-TT Live documents into EBU-TT-D documents that can be later used to be embedded in video streams such as DASH.
 The script launches with the ``ebu-ebuttd-encoder`` command. There are switches to control the mediatime conversion
-reference point and the segmentation interval. Please run ``ebu-ebuttd-encoder --help`` to find our more.
\ No newline at end of file
+reference point and the segmentation interval. Please run ``ebu-ebuttd-encoder --help`` to find our more.
+
+The ebu-run script
+------------------
+This script is capable of picking up a structured configuration file and use that to create the nodes and
+carriage mechanism objects that we would like to use and they can be even wired in the same configuration file
+together so in the end a complex system can be modeled using a single json file. Please refer to
+:py:mod:`ebu_tt_live.scripts.ebu_run` as well as :py:mod:`ebu_tt_live.config` to learn more about the
+way the configuration logic works. To get help on permitted options please run ``ebu-run --help``.
```

Comment:
Hm I should have elaborated more there... Since there is no node type specified the logic cannot decide what options it should give you. I did not get into giving it extra functionality to force it to list each type and then each type carriage and each type of everything else.

If it did it automatically on its own it would list everything exponential times :)